libs/ledger-live-common/src/families/filecoin/bridge/utils/addresses.ts

```diff
@@ -1,150 +1,126 @@
+import blake from "blakejs";
 import base32Decode from "base32-decode";
-import { blake2bInit, blake2bUpdate, blake2bFinal, Blake2bCTX } from "blakejs";
-import leb128 from "leb128";
-import { log } from "@ledgerhq/logs";
 
-type PrefixObjType = Readonly<{ f: "mainnet" } & { t: "testnet" }>;
-type PrefixType = keyof PrefixObjType;
+// @ts-ignore
+import BN from "bn.js";
```

Comment:
Does our Bignumber lib that we are already using in live doesn't work ? 
+ the @ts-ignore would not be acceptable

Reply:
This is what we cannot do with BigNumber.js 
https://github.com/Zondax/ledger-live/blob/0f4fee91f4ca1227164df4df09f7e9d7bdc36cc1/libs/ledger-live-common/src/families/filecoin/bridge/utils/addresses.ts#L111

The ts-ignore is because of the lack of definition file.